### DISCUSSION TITLE
KeyError: "Registering two gradient with name 'Gelu'!

### DISCUSSION THREAD

**Discussion by wankiwi** [deepmodeling/deepmd-kit#1371] [Q&A]
Hi, everyone.
I installed Deepmd-kit 2.0.3 successfully from the source code without any error message. But when I tried to train a model, I got the following error message.

`**KeyError: "Registering two gradient with name 'Gelu'! (Previous registration was in register /public/software/apps/DeepLearning/Tensorflow/tensorflow_2.3.3-rocm_3.9.1/lib/python3.6/site-packages/tensorflow/python/framework/registry.py:66)"**`

How to solve it? Thanks in advance.
PS: my Tensorflow version is tensorflow_rocm-2.3.3 on a machine with dcu 
[slurm-236155.zip](https://github.com/deepmodeling/deepmd-kit/files/7760756/slurm-236155.zip)

**Comment by wanghan-iapcm** [comment]
@denghuilu could you try to reproduce this error? Thanks!
